samples/msal-node-samples/custom-INetworkModule-and-network-tracing/app.ts

```diff
@@ -4,7 +4,7 @@ import * as msal from "@azure/msal-node";
  * After "npx tsc" is executed via the "npm run start" script, app.ts and HttpClient.ts are compiled to .js and stored in the /dist folder
  * The app is run via "node dist/app.js", hence the .js import of the HttpClient
  */
-// import HttpClient from "./HttpClient.js";
+// import { HttpClient } from "./HttpClient.js";
 
 const clientConfig: msal.Configuration = {
     auth: {
```

Comment:
line 41 below:

`networkClient: new class CustomHttpClient implements INetworkModule`

should be 

`networkClient: new class CustomHttpClient implements msal.INetworkModule`